source/brailleDisplayDrivers/nattiqbraille.py

```diff
@@ -0,0 +1,140 @@
+# brailleDisplayDrivers/nattiqbraille.py
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2011-2020
+
+
+import serial
+import braille
+import inputCore
+from logHandler import log
+import hwIo
+
+BAUD_RATE = 10000000
+TIMEOUT = 0.3

```

Comment:
Rather than a constant, please use the timeout property on the driver. See the handy tech driver for an example.

Reply:
This is done for all timeout mentions